temple/update.py

```diff
@@ -34,13 +34,17 @@ def _cookiecutter_configs_have_changed(template, old_version, new_version):
     Returns:
         bool: True if the cookiecutter.json files have been changed in the old and new versions
     """
-    with tempfile.TemporaryDirectory() as clone_dir:
-        repo_dir = cc_vcs.clone(template, old_version, clone_dir)
-        old_config = json.load(open(os.path.join(repo_dir, 'cookiecutter.json')))
-        subprocess.check_call(f'git checkout {new_version}', cwd=repo_dir, shell=True, stderr=subprocess.PIPE)
-        new_config = json.load(open(os.path.join(repo_dir, 'cookiecutter.json')))
+    temple.check.is_git_ssh_path(template)
+    repo_path = temple.utils.get_repo_path(template)
+    github_client = temple.utils.GithubClient()
+    api = '/repos/{}/contents/cookiecutter.json'.format(repo_path)
+
+    old_config_resp = github_client.get(api, params={'ref': old_version})
+    old_config_resp.raise_for_status()
+    new_config_resp = github_client.get(api, params={'ref': new_version})
+    new_config_resp.raise_for_status()
 
-    return old_config != new_config
+    return old_config_resp.json()['content'] != new_config_resp.json()['content']
```

Comment:
This seems to reverse the change I made to remove use of the github api?

Reply:
I was deluded into thinking it still worked - and I'm not sure why it is when I bring back their GitHub client - which always calls the API